website/source/intro/getting-started/running.html.md

```diff
@@ -40,27 +40,47 @@ vagrant@nomad:~$ sudo nomad agent -dev
+    2016/08/23 21:26:03.854711 [INFO] client: using state directory /tmp/NomadClient632906511
+    2016/08/23 21:26:03.854732 [INFO] client: using alloc directory /tmp/NomadClient174428962
+    2016/08/23 21:26:03.854761 [DEBUG] client: built-in fingerprints: [arch cgroup cpu env_aws env_gce host memory network nomad storage]
+    2016/08/23 21:26:03.854879 [INFO] fingerprint.cgroups: cgroups are available
+    2016/08/23 21:26:03.854951 [DEBUG] fingerprint.cpu: frequency: 2294 MHz
+    2016/08/23 21:26:03.854954 [DEBUG] fingerprint.cpu: core count: 1
+    2016/08/23 21:26:03 [INFO] raft: Node at 127.0.0.1:4647 [Follower] entering Follower state (Leader: "")
+    2016/08/23 21:26:03.861876 [INFO] nomad: adding server nomad.global (Addr: 127.0.0.1:4647) (DC: dc1)
+    2016/08/23 21:26:03.861924 [DEBUG] client: fingerprinting cgroup every 15s
+    2016/08/23 21:26:05 [WARN] raft: Heartbeat timeout from "" reached, starting election
+    2016/08/23 21:26:05 [INFO] raft: Node at 127.0.0.1:4647 [Candidate] entering Candidate state
+    2016/08/23 21:26:05 [DEBUG] raft: Votes needed: 1
+    2016/08/23 21:26:05 [DEBUG] raft: Vote granted from 127.0.0.1:4647. Tally: 1
+    2016/08/23 21:26:05 [INFO] raft: Election won. Tally: 1
+    2016/08/23 21:26:05 [INFO] raft: Node at 127.0.0.1:4647 [Leader] entering Leader state
+    2016/08/23 21:26:05 [INFO] raft: Disabling EnableSingleNode (bootstrap)
+    2016/08/23 21:26:05 [DEBUG] raft: Node 127.0.0.1:4647 updated peer set (2): [127.0.0.1:4647]
+    2016/08/23 21:26:05.685611 [INFO] nomad: cluster leadership acquired
+    2016/08/23 21:26:05.685685 [DEBUG] leader: reconciling job summaries at index: 0
+    2016/08/23 21:26:05.855402 [DEBUG] fingerprint.env_aws: Error querying AWS Metadata URL, skipping
+    2016/08/23 21:26:07.855934 [DEBUG] fingerprint.env_gce: Could not read value for attribute "machine-type"
+    2016/08/23 21:26:07.855942 [DEBUG] fingerprint.env_gce: Error querying GCE Metadata URL, skipping
+    2016/08/23 21:26:07.860576 [DEBUG] fingerprint.network: Detected interface lo with IP 127.0.0.1 during fingerprinting
+    2016/08/23 21:26:07.861368 [DEBUG] fingerprint.network: Unable to read link speed from /sys/class/net/lo/speed
+    2016/08/23 21:26:07.861376 [DEBUG] fingerprint.network: Unable to read link speed; setting to default 100
+    2016/08/23 21:26:07.863896 [DEBUG] client: applied fingerprints [arch cgroup cpu host memory network nomad storage]
+    2016/08/23 21:26:07.863914 [DEBUG] driver.exec: exec driver is enabled
+    2016/08/23 21:26:07.864114 [DEBUG] client: fingerprinting exec every 15s
+    2016/08/23 21:26:07.864966 [DEBUG] driver.docker: using client connection initialized from environment
+    2016/08/23 21:26:07.867270 [DEBUG] client: available drivers [exec raw_exec docker]
+    2016/08/23 21:26:07.868498 [DEBUG] client: fingerprinting docker every 15s
+    2016/08/23 21:26:07.872181 [DEBUG] client: node registration complete
```

Comment:
Yea I kind of figured that. Is it worth making a note regarding the error related to automatic consul registration?